### PR TITLE
update: SQL application to restart strategy

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/flink/howto/restart-strategy-jar-applications.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-Aiven for Apache Flink® includes a default restart strategy for JAR
-applications. This strategy uses the **exponential-delay** technique,
-incrementally increasing the delay time between restarts up to a
-specified maximum. Once this maximum delay is reached, it remains
-constant for any subsequent restarts. The default strategy is fully
-integrated into the Aiven for Apache Flink cluster configuration and
+## Restart strategy for JAR applications
+
+Aiven for Apache Flink® uses the **exponential-delay** as the default restart strategy
+for JAR applications, incrementally increasing the delay time between restarts to a
```
[the remark this answers is not in the document]
Comment (MichaelTansiniAiven) [reply docs/products/flink/howto/restart-strategy-jar-applications.md:17]: Perhaps we can pull this paragraph into the default restart strategy and then explain how to alter it for JARs, and then note in SQL we can only turn it on/off

[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/restart-strategy-jar-applications.md]: Updated the content and structure.

File: docs/products/flink/howto/create-jar-application.md [hunk 2]
```diff
@@ -3,28 +3,19 @@ title: Create a JAR application
...
-    1. Click **Service settings** on the left sidebar.
-    1. Scroll to the **Advanced configuration** section, and click
-        **Configure**.
-    1. In the **Advanced configuration** screen, click **Add
-       configuration options**, and using the search box, find and set
-       `custom_code` configuration to **Enabled**.
+  To try this feature, request access by contacting the [sales team](mailto:sales@aiven.io).
+- Once activated on your Aiven account, create a new Aiven for Apache Flink service and
+toggle the option for **Upload and deploy custom JARs** to enable custom JARs during
+service creation.
```
Comment (ArthurFlag) [inline docs/products/flink/howto/create-jar-application.md:14]: indention issue :) 

We can do a bit shorter with `click **Upload and deploy custom JARs** to enable` or `toggle **Upload and deploy custom JARs**`, no need to say it is an option, I'd argue?

File: docs/products/flink/howto/create-jar-application.md [hunk 3]
```diff
@@ -54,7 +51,8 @@ processing capabilities.
         a specific state. No
         savepoints are available for the first application deployment.
     1.   Toggle **Restart on failure** to automatically restart Flink
-        jobs upon failure.
+        jobs upon failure. See [Restart strategy in SQL and JAR applications](/docs/products/flink/howto/restart-strategy-jar-applications)
+        for details.
```
Comment (ArthurFlag) [inline docs/products/flink/howto/create-jar-application.md:55]: indention seems off in the list?

File: docs/products/flink/howto/create-sql-application.md [hunk 4]
```diff
@@ -51,32 +51,32 @@ Console](https://console.aiven.io/):
...
         the sink table.
     -   Click **Add table**.
 
-10. Click **Next** to enter the **SQL statement** that transforms the
+1. Click **Next** to enter the **SQL statement** that transforms the
     data from the source stream. Optionally, click **Run** to see how
     the data is extracted from the source.
 
-11. Click **Save and deploy later** to save the application. You can
+1. Click **Save and deploy later** to save the application. You can
```
Comment (ArthurFlag) [inline docs/products/flink/howto/create-sql-application.md:73]: indention for former double digits is off

**Review by ArthurFlag — APPROVED** [review]
🙌

**Pull request merged**